SBTi/data/__init__.py

```diff
@@ -2,12 +2,59 @@
 This module contains classes that create connections to data providers.
 """
 import logging
-from typing import Type
+from typing import Type, List, Dict
 
 import pandas as pd
 
 from SBTi.configs import ColumnsConfig
 
+from .data_provider import DataProvider
+from .csv import CSVProvider
+from .excel import ExcelProvider
+from .bloomberg import Bloomberg
+from .cdp import CDP
+from .iss import ISS
+from .trucost import Trucost
+from .urgentum import Urgentum
+
+
+DATA_PROVIDER_MAP: Dict[str, Type[DataProvider]] = {
```

Comment:
hmm, maybe use a class instead so you have this map fully typed (incl. the keys)?